packages/design-system/src/components/Form/FieldGroup/FieldGroup.tsx

```diff
@@ -41,24 +46,27 @@ const FieldGroup = React.forwardRef(
 
 		const focusField = () => fieldRef.current?.focus();
 
-		const Description = () => {
+		const Description = ({ desc }: { desc: string }) => {
 			const descriptionProps = {
 				small: true,
-				description,
+				description: desc,
 			};
+			if (!description) {
+				return null;
+			}
 			if (hasError) {
-				return <InlineMessage.Destructive {...descriptionProps} />;
+				return <InlineMessageDestructive {...descriptionProps} />;
 			}
 			if (hasWarning) {
-				return <InlineMessage.Warning {...descriptionProps} />;
+				return <InlineMessageWarning {...descriptionProps} />;
 			}
 			if (hasSuccess) {
-				return <InlineMessage.Success {...descriptionProps} />;
+				return <InlineMessageSuccess {...descriptionProps} />;
 			}
 			if (hasInformation) {
-				return <InlineMessage.Information {...descriptionProps} />;
+				return <InlineMessageInformation {...descriptionProps} />;
 			}
```

Comment:
`variant` could be used here 👼

Reply:
Kinda not though, we'd have to map the various `hasState` to `variant` props. That's sort of the same code down the line.